justifi-tech/web-component-library
# feat: Business Details

* Initial HTML structure
* Fetches data
* Handles loading state

<img width="1345" alt="image" src="https://github.com/justifi-tech/web-component-library/assets/22480988/be0e2cb1-4123-4591-9254-973894798a8d">






Links
-----



Developer considerations
--------------

  - If this is a MAJOR version, what are the breaking API changes? Was anything added to the API?
  - If this is a MINOR version, what feature was added? Why would consumers want this new version?
  - If this is a PATCH version, what was fixed?


Steps for Testing/QA
--------------------

* Go to http://localhost:6006/?path=/story/components-businessdetails--basic
* Add the `auth-token` and a `business-id` props
* If wrong token or ID is passed an error must appear
* If the right props are passed the right data must be displayed
